docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md

```diff
@@ -28,6 +28,14 @@ For instructions on how to add a data source to Grafana, refer to the [administr
 Only users with the organization `administrator` role can add data sources.
 Administrators can also [configure the data source via YAML](#provision-the-data-source) with Grafana's provisioning system.
 
+## Configuring Elasticsearch
```

Comment:
```suggestion
## Configuring Elasticsearch permissions
```

or just

```suggestion
## Configuring permissions
```

Maybe something like this? The first heading already says `Configure the Elasticsearch data source`. Would be weird to have another `Configuring Elasticsearch` section there in my opinion.

Reply:
👍 yeah, I think  `Configuring permissions` make sense and is clear.